go.mod

```diff
@@ -5,6 +5,7 @@ go 1.22.0
 toolchain go1.22.1
 
 require (
+	github.com/adhocore/gronx v1.8.1
```

Comment:
@soltysh mentioned we can also use a `DelayQueue` from k8s itself, together with a cron parser. Can you check whether this would be a viable alternative here?

Reply:
👍🏽